m2m/m2m.py

```diff
@@ -147,75 +148,74 @@ def writeTemplateFiles(self, baseDirectory, foldername):
         writeDirectory = os.path.join(baseDirectory, foldername)
         try:
             os.makedirs(writeDirectory)
-        except:
+        except OSError:
             if os.path.exists(writeDirectory):
                 pass  # no big deal if they exist
             else:
                 raise MetadataConverterException(
-                    "Unable to create the output directory '%s'. " +
-                    "Perhaps you should check permissions?" %
-                    (writeDirectory,))
-        with open(os.path.join(writeDirectory, "metadata.xml"), "wb") as templateFile:
+                    'Unable to create the output directory %s. ' +
+                    'Perhaps you should check permissions?' %
+                    writeDirectory)
+        with open(os.path.join(writeDirectory, 'metadata.xml'), 'wb') as templateFile:
             templateFile.write(self.__bytes__())
-        return "%s finished" % foldername
+        return '%s finished' % foldername
 
     def writeJSONFile(self, baseDirectory, foldername, data):
         baseDirectory = baseDirectory
         foldername = foldername
         writeDirectory = os.path.join(baseDirectory, foldername)
         try:
             os.makedirs(writeDirectory)
-        except:
+        except OSError:
             if os.path.exists(writeDirectory):
                 pass  # no big deal if they exist
             else:
                 raise MetadataConverterException(
-                    "Unable to create the output directory '%s'. " +
-                    "Perhaps you should check permissions?" %
-                    (writeDirectory,))
-        jsonFilename = open(os.path.join(writeDirectory, "metadata.json"), "w")
+                    'Unable to create the output directory %s.'
+                    'Perhaps you should check permissions?' % writeDirectory)
+        jsonFilename = open(os.path.join(writeDirectory, 'metadata.json'), 'w')
         jsonFile = json.dumps(data,
                               sort_keys=True,
                               indent=4,
                               separators=(',', ': ')
                               )
         jsonFilename.write(jsonFile)
         jsonFilename.close()
```

Comment:
This could be:
```
        with open(os.path.join(writeDirectory, 'metadata.json'), 'w') as jsonFilename:
            jsonFilename.write(json.dumps(data,
                               sort_keys=True,
                               indent=4,
                               separators=(',', ': ')))

```

Reply:
sure!